src/main/java/com/example/Restaurant_Management_System_REST_API/DTO/CustomerDTOs/CustomerDTOResponse.java

```diff
@@ -0,0 +1,34 @@
+package com.example.Restaurant_Management_System_REST_API.DTO.CustomerDTOs;
+
+import com.example.Restaurant_Management_System_REST_API.model.ContactDetails;
+import com.example.Restaurant_Management_System_REST_API.model.entity.Authority;
+import com.example.Restaurant_Management_System_REST_API.model.entity.Reservation;
+import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Data
+public class CustomerDTOResponse {
+
+
+    private Long id;
+    private LocalDateTime creationTime;
+    private Reservation reservation;
+    private ContactDetails contactDetails;
+
+    //Below are fields for security issues
+    private String password;
```

Comment:
no, you are right! especially that password is being hashed!

Reply:
Oh now I remember, if I delete password field from here then I'm not able to update password in update():

![image](https://github.com/BartoliniAndBorderCollies/Restaurant_Management_System_REST_API/assets/126821059/ae3f544d-77d1-4f5f-81f1-46f14e3790ac)

Reply:
Yep, I see. But you can easily overcome this issue. Do you have any ideas how this could be done?

Reply:
I have some idea, what do you think about this?

![image](https://github.com/BartoliniAndBorderCollies/Restaurant_Management_System_REST_API/assets/126821059/2a7504fd-7ab5-4331-912e-da4a4edadba1)

Reply:
and also I would need to change integration tests because now I check if the password provided is the same as it was returned. for example here:

![image](https://github.com/BartoliniAndBorderCollies/Restaurant_Management_System_REST_API/assets/126821059/16fe2871-bda8-4df2-bae8-42d3a1fe0344)

Reply:
and this is how could I remake the integration test to still check if password is correct. tell me what you think about this:

![image](https://github.com/BartoliniAndBorderCollies/Restaurant_Management_System_REST_API/assets/126821059/8d727a4e-8592-4948-9513-201b8cfc386e)

Reply:
or I could resign totally from checking the passwords -> I dont know how it is in real commercial projects..

Reply:
What you had in the beginning was okay. The problem was only to return password as a response, especially in the encoded form as this makes whole password security vulnerable.
I'd suggest simply omitting password in response :)